data-products/docs/flows/sql_etl/usage.md

```diff
@@ -0,0 +1,145 @@
+# Usage
+
+`sql_etl` is a Prefect flow meant to act as a service for executing simple SQL based extract and load workflows.  This means a SQL statement extracts the data to cloud storage.  Then a SQL statement loads this data into a cloud-based Data Warehouse.  That is the expectation for how this should be used.
+
+This is technically more of an `el`, but `etl` is a more common term to use.
+
+A `sql_etl` job can be incremental and use offset logic or simply just run an extract-load process without worring about tracking offset or batching by day.
+
+Code lives at `src/sql_etl` and is described [here](/flows/sql_etl/code).
+
+## Expectations
+
+To leverage this service for jobs, `sql_etl` you will need to know:
+
+- How to create the SQL expected for things to work
+- How the offset and interval logic works
+- How to setup your local environment
+- How to create a deployment
+
+Below we go through each element.
+
+
+## SQL statments
+
+SQL statments will live in subfolders at `src/sql_etl/sql`
+
+Here is an example of what a set of sql job files looks like:
+
+```
+sql_etl
+└── sql
+    ├── backend_events_for_mozilla
+    │   └── data.sql
+    ├── curated_feed_exports_aurora
+    │   ├── curated_feed_items
+    │   │   ├── data.sql
+    │   │   └── load.sql
+    │   ├── curated_feed_items_deleted
+    │   │   ├── data.sql
+    │   │   └── load.sql
+    │   ├── curated_feed_prospects
+    │   │   ├── data.sql
+    │   │   └── load.sql
+    │   ├── curated_feed_queued_items
+    │   │   ├── data.sql
+    │   │   └── load.sql
+    │   └── tile_source
+    │       ├── data.sql
+    │       └── load.sql
+    ├── firefox_new_tab_impressions
+    │   ├── firefox_new_tab_daily_disable_rate_by_feed
+    │   │   ├── data.sql
+    │   │   └── offset.sql
+    │   ├── firefox_new_tab_daily_spoc_fill_rate_by_position_feed
+    │   │   ├── data.sql
+    │   │   └── offset.sql
+    │   ├── firefox_new_tab_daily_unique_engagement_by_feed
+    │   │   ├── data.sql
+    │   │   └── offset.sql
+    │   └── firefox_new_tab_monthly_unique_engagement_by_feed
+    │       ├── data.sql
+    │       └── offset.sql
+    └── impression_stats_v1
+        ├── data.sql
+        ├── load.sql
+        └── offset.sql
+```
+
+Notice that we can have up to three files possible in a folder:
+
+- `data.sql`
+- `load.sql`
+- `offset.sql`
```

Comment:
Could you add [the flow chart](https://user-images.githubusercontent.com/11067272/256566098-8fa32b7b-d325-44b0-a0ae-a303b4f4160c.png) you made here?